task/style.css

```diff
@@ -279,3 +276,148 @@ footer {
   border-radius: 50%;
   margin: 0 1rem;
 }
+/* Skills field */
+/* Multi-select container styles */
+.multi-select-container {
+  position: relative;
+  width: 100%;
+}
+
+.multi-select-button {
+  display: flex;
+  align-items: center;
+  min-height: 48px;
```

Comment:
- as the height of the button will not be changed let's use `height` instead of min height.

Reply:
This is different from the regular button... we show "Skill badges" on button which can increase the height depending on number of badges